src/broker/broker_admin_pub.c

```diff
@@ -324,9 +326,17 @@ admin_start_cmd (T_BROKER_INFO * br_info, int br_num, int master_shm_id, bool ac
     }
   chdir (envvar_bindir_file (path, BROKER_PATH_MAX, ""));
 
+  if (gethostname (hostname, sizeof (hostname)) < 0)
```

Comment:
This is for the line 'hostname' of line 337.
If this line is not written, 'hostname' has no any chacters.